### PR TITLE
Restore "Generate prisma client outside of node modules, cache generated client"

### PR DESCRIPTION
**Pull request by t3dotgg**
un-reverts t3-oss/create-t3-turbo#37

First we should fix https://github.com/t3-oss/create-t3-turbo/issues/42

### CODE REVIEW

**Comment by juliusmarminge** [comment]
As long as my execution hours on railway doesn't run out the test to verify is that the posts are loading on the preview deployment

File: turbo.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "pipeline": {
+    "db-generate": {
+      "outputs": [".generated/**"]
+    },
```
Comment (timbuckley) [inline turbo.json:5]: I'm curious, why bother hoisting `db-generate` to the top here?

Comment (TheoBr) [reply turbo.json:5]: I like having this read "in the order things happen", the other order felt arbitrary

**Review by fysp11 — APPROVED** [review]
Good step in the right direction :)

**Comment by juliusmarminge** [comment]
Don't think this is possible until Prisma fixes their beloved relationship with the node_modules.

@mattpocock attempted to fix this for the [turborepo docs](https://turbo.build/repo/docs/handbook/prisma#7-caching-the-results-of-prisma-generate) without success.

**Comment by mattpocock** [comment]
Yeah, this doesn't work when deployed to Vercel, at least.

**Pull request closed without merging**